### DISCUSSION TITLE
1-1-0 fluid requester and provider are too large

### DISCUSSION THREAD

**Discussion by cyborg728** [Opinionated-Blueprints/10-Books-Full-of-Rails#72] [Ideas]
Hello.
1-1-0 item providers is not in 42x42 grid. Why? Why do you need 3 extra tanks? Please add 1-1-0 fluid requester and provider 42x42 size.
And add a crypto wallet or paypal for donations. I don't want to link my card to an unknown service.
![image](https://github.com/user-attachments/assets/79ac0793-27cf-4596-8ea7-66a02cfc2693)
![image](https://github.com/user-attachments/assets/ccc4c448-4077-4d26-a4cb-dc1f608b4d74)

**Comment by cyborg728** [comment]
and need some fix 
![image](https://github.com/user-attachments/assets/ba21e665-e0e0-4a9d-aab2-f3439b5b3967)

**Comment by cyborg728** [comment]
ok i see that pupm can't connect to fluid wagon that is not on straight rails 😒

**Comment by Bocian-1** [comment]
The grid is 14x14 which is a third of 42x42 so it still aligns. This is to make it compatible with the hypermodular rail blueprints which are also on 14x14. It's also better for 42x42 since you can place it more precisely and bridge any gap with hypermodular if needed.
The additional tanks are just because they fit at this point since the size is already at the minimum due to the pump not being able to connect if the train isn't perfectly straight or on [perfectly calculated turns](https://www.reddit.com/r/technicalfactorio/comments/1gvyzv8/new_rails_new_rail_lengths_reverse_engineering/).

PS; I've made a PayPal account and connected it to Ko-Fi. It still needs some verification step that may take a few days apparently so I'm not sure if it will accept anything quite yet.

**Reply by cyborg728** [reply]
crypto wallet

**Reply by Bocian-1** [reply]
ok... I made a wallet with brave. I'll admit I have no idea how any of this works in practice, but here are all the addresses I got out of it:
0xB06BeF23eb15348eb86f097780374DE3D6743b65
f13v4pavbfldykbh3fufem2g2a7hujcxqumcos56a
2NxV6TXYSwuYMxs7TariegGWT9M9ZJCUw8BwGJfzFXWd

**Comment by cyborg728** [comment]
do i understand correctly that for each group i need to create refueling station? please explain to me why? there is only 4 type of fuel. i think as the game progress, all trains will use the same fuel.

**Reply by Bocian-1** [reply]
Yes, you need a refuelling station for each group. The fuel types have nothing to do with it. You can't tell what configuration a train is so I can't make a universal refuelling station like it was possible with LTN and they have to be tailored to specific train configurations so it doesn't load fuel into a cargo wagon. You won't need any more than the number of configurations as long as you keep all trains of each configuration in the same group and there isn't really a reason not to, though you do need to separate solid and liquid cargo into different groups and have separate refuelling for them.

**Reply by OldieButGoldie** [reply]
>  though you do need to separate solid and liquid cargo into different groups and have separate refuelling for them.

A little offtopic about this: I have actually been using solid and liquid trains with the same (n of locomotives-n of wagons) configurations, and assigned them the same group (mainly because I am still rocking base game, haven't yet bought the DLC, so no quality mod here). It works because a liquid train only visits stations called `<group> <provider> <fluid parameter wildcard>` and then `<group> <requester> <required fluid>`. The different names of the stations make the system work flawlessly!

**Comment by cyborg728** [comment]
pls check outpost stackers
![image](https://github.com/user-attachments/assets/4a316ff3-e406-45a0-a13f-9100a6e6c233)
![image](https://github.com/user-attachments/assets/2e1fe09f-d01d-4a42-9049-afe14d427e2f)
There is crooked turn

**Reply by Bocian-1** [reply]
They're not really supposed to be aligned with u-turns. U-turns are 1 rail tile wider than they need to be for symmetry. Outpost stackers are optimised for space efficiency
